gno.land/cmd/gnoland/testdata/event.txtar

```diff
@@ -0,0 +1,60 @@
+# start a new node
+gnoland start
+
+gnokey maketx addpkg -pkgdir $WORK -pkgpath gno.land/r/demo/ee -gas-fee 1000000ugnot -gas-wanted 2500000 -broadcast -chainid=tendermint_test test1
+stdout OK!
+stdout 'GAS WANTED: 2500000'
+stdout 'GAS USED: [0-9]+'
+stdout 'EVENTS: \[\]'
+
+gnokey maketx call -pkgpath gno.land/r/demo/ee -func Sender -gas-fee 1000000ugnot -gas-wanted 2000000 -broadcast -chainid=tendermint_test test1
+stdout OK!
+stdout 'GAS WANTED: 2000000'
+stdout 'GAS USED: [0-9]+'
+stdout 'EVENTS: \[?.*sender.*SubSender.*receiver.*SubReceiver.*\]'
```

Comment:
can this be more explicit too?

Reply:
Yes, please provide the complete JSON data and use a regular expression for the timestamp (until we remove it). Apply the same approach to all other text areas.

Reply:
I updated the regex this way:
1. make capture group for the part of the JSON that represents each event
2. check if it repeats more than once (also, comma-sperated)
```regex
\[({\"type\":\"\w+\",\"pkg_path\":\"?.*\",\"identifier\":\"\w+\",\"attributes\":\[{?.*}\]})(,\{"type":"\\w+","pkg\\_path":".*?","identifier":"\\w+","attributes":\[.*?\]\})*\]
```

Reply:
Why are you using `\w+` everywhere? Is this test consistent, except for the timestamp that needs removal?

Reply:
Check to get the exact same string.

https://github.com/gnolang/gno/pull/1653/commits/150fe460a513dbed38cd41213ed49ab681c809cb